sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/SqlOperators.java

```diff
@@ -266,7 +275,7 @@ public SqlSyntax getSyntax() {
     };
   }
 
-  private static RelDataType createSqlType(SqlTypeName typeName, boolean withNullability) {
+  public static RelDataType createSqlType(SqlTypeName typeName, boolean withNullability) {
```

Comment:
Why does this need to be public?

Reply:
It was mistakenly done, reverted it to the private.